app/controllers/api/physical_servers_controller.rb

```diff
@@ -40,6 +40,24 @@ def restart_mgmt_controller_resource(type, id, _data)
       change_resource_state(:restart_mgmt_controller, type, id)
     end
 
+    def refresh_physical_server(physical_server)
```

Comment:
maybe move this to private section

Reply:
@abellotti @jntullo Thanks for your feedback, just fixed it.